build/jobs/run-pr-tests.yml

```diff
@@ -98,6 +98,15 @@ jobs:
         Write-Host "##vso[task.setvariable variable=TestContainerRegistryServer]$($acrLoginServer)"
         Write-Host "##vso[task.setvariable variable=TestContainerRegistryPassword]$($acrPassword)"
 
+        $exportStoreSettings = $appSettings | where {$_.Name -eq "FhirServer__Operations__Export__StorageAccountUri"}
```

Comment:
[https://github.com/microsoft/fhir-server/blob/de304363c71749dc993f6b5fa9076b55b6561e32/samples/templates/default-azuredeploy.json#L235](https://github.com/microsoft/fhir-server/blob/de304363c71749dc993f6b5fa9076b55b6561e32/samples/templates/default-azuredeploy.json#L235)
or 
[https://github.com/microsoft/fhir-server/blob/de304363c71749dc993f6b5fa9076b55b6561e32/samples/templates/default-azuredeploy-docker.json#L231](https://github.com/microsoft/fhir-server/blob/de304363c71749dc993f6b5fa9076b55b6561e32/samples/templates/default-azuredeploy-docker.json#L231)

need to be updated to include 
FhirServer__Features__SupportsAnonymizedExport

Reply:
yes, this option previous major to disable for PaaS. Update the default value to true.